bin/generate_sdk_documentation.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import docstring_parser
+import inspect
+from pathlib import Path
+import sys
+
+from substra import Client
+from substra.sdk.utils import retry_on_exception
+
+
+MODULE_LIST = [
+    Client,
+    retry_on_exception
+]
+
+KEYWORDS = [
+    "Args",
+    "Returns",
+    "Yields",
+    "Raises",
+    "Example"
+]
+
+
+def generate_function_help(fh, asset):
+    """Write the description of a function"""
+    fh.write(f"# {asset.__name__}\n")
+    signature = str(inspect.signature(asset))
+    fh.write("```python\n")
+    fh.write(f"{asset.__name__}{signature}")
+    fh.write("\n```")
+    fh.write("\n\n")
+    # Write the docstring
+    docstring = inspect.getdoc(asset)
+    docstring = docstring_parser.parse(inspect.getdoc(asset))
+    fh.write(f"{docstring.short_description}\n")
+    if docstring.long_description:
+        fh.write(f"{docstring.long_description}\n")
+    # Write the arguments as a list
+    if len(docstring.params) > 0:
+        fh.write("\nArguments:\n")
```

Comment:
Could you make arguments and returns bold? Making them into level3 headers may be too much, but a bit of bold would be nice:)